src/actions/auth.js

```diff
@@ -345,14 +345,14 @@ const handleAuthStateChange = (dispatch, firebase, authData) => {
       setupPresence(dispatch, firebase)
     }
 
-    watchUserProfile(dispatch, firebase)
-
```

Comment:
I've just change the order of these two function. The result is that the LOGIN action will be called before SET_PROFILE as it should be.